### PR TITLE
Fix queryFunc if adlists URLs have been removed

### PR DESCRIPTION
**Pull request by WaLLy3K**
**By submitting this pull request, I confirm the following:**

- [x] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md).
- [x] I have checked that [another pull request](https://github.com/pi-hole/pi-hole/pulls) for this purpose does not exist.
- [x] I have considered, and confirmed that this submission will be valuable to others.
- [x] I accept that this submission may not be used, and the pull request closed at the will of the maintainer.
- [x] I give this submission freely, and claim no ownership to its content.

**How familiar are you with the codebase?:**

10

---

* Allow for -adlists command line switch (where the "s" is a typo)
* Add error message when unable to find associated adlists URL

### CODE REVIEW

File: pihole [hunk 1]
```diff
@@ -535,7 +541,6 @@ Switch Pi-hole subsystems to a different Github branch
 Repositories:
   core [branch]       Change the branch of Pi-hole's core subsystem
   web [branch]        Change the branch of Admin Console subsystem
-  ftl [branch]        Change the branch of Pi-hole's FTL subsystem
```
Comment (Mcat12) [inline pihole]: Why is this deleted?

File: pihole [hunk 2]
```diff
@@ -429,15 +435,15 @@ Time:
     # Enable Pi-hole
     echo -e "  ${INFO} Enabling blocking"
     local str="Pi-hole Enabled"
-
+
```
Comment (Mcat12) [inline pihole]: Lots of extra whitespace

File: pihole (hunk 1 above)
Comment (WaLLy3K) [inline pihole]: Oh shoot, I edited my local `pihole` and not from the latest Development. I have no idea how I missed that :sweat_smile:

**Comment by PromoFaux** [comment]
Approved
